misc/graylog.conf

```diff
@@ -394,8 +394,8 @@ stream_aware_field_types=false
 #   - delete # Deletes the index completely (Default)
 #   - close # Closes the index and hides it from the system. Can be re-opened later.
 #   - none #  No operation is performed. The index stays open. (Not recommended)
-# WARNING: At least one strategy must be enabled
-disabled_retention_strategies = none
+# WARNING: At least one strategy must be enabled. Only intended for new installs!
```

Comment:
I know what you are trying to say here, but I guess most users won't understand it.
How about
```suggestion
# WARNING: At least one strategy must be enabled. Be careful when extending this list on existing installations!
```